modules/installation-aws-iam-policies-about.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-account.adoc
+
+:_content-type: CONCEPT
+[id="iam-policies-and-aws-authentication_{context}"]
+= IAM Policies and AWS authentication
+
+By default, the installation program creates instance profiles for the bootstrap, control plane, and compute instances with the necessary permissions for the cluster to operate.
+
+However, you can create your own IAM roles and specify them as part of the installation process. You may need to specify your own roles to deploy the cluster or to manage the cluster after installation. For example:
```

Comment:
ISG: Consider using 'might or 'can' rather than 'may'.

```suggestion
However, you can create your own IAM roles and specify them as part of the installation process. You might need to specify your own roles to deploy the cluster or to manage the cluster after installation. For example:
```

Reply:
Also yes. Thank you.